core/src/components/button/button.ionic.scss

```diff
@@ -36,6 +36,12 @@
     z-index: 1;
   }
 
+  ::slotted(ion-icon[slot="start"]),
+  ::slotted(ion-icon[slot="end"]),
+  ::slotted(ion-icon[slot="icon-only"]) {
+    font-size: px-to-rem(16);
+  }
```

Comment:
I noticed that there are two instances of this. What's the difference between this one and the [grouping](https://github.com/ionic-team/ionic-framework/blob/3c1936fe4398f8be0b93f960832c076df27564e9/core/src/components/button/button.ionic.scss#L115-L119) further in the file?

Reply:
You're right @thetaPC 🤦. Thank you!
Already fixed!